browser/src/layer/marker/TextInput.js

```diff
@@ -439,8 +439,8 @@ L.TextInput = L.Layer.extend({
 				_('Screen reader support for text content is disabled. ') +
 				_('You need to enable it both at server level and in the UI. ') +
 				_('Look for the accessibility section in coolwsd.xml for server setting. ') +
-				_('Also check the voice over toggle under %parentControl.').replace(
-					'%parentControl',
+				_('Also check the voice over toggle under {parentControl}.').replace(
+					'{parentControl}',
```

Comment:
Fine, but what's the difference between %foo and {foo} in this case?

If we want to be consistent, perhaps just go with {0}?

Reply:
PO processing tools choked on %foo, we had complaints in forums. Somehow they were mistreated as printf format strings.
We have a lot of non-{0} style placeholders in the code base, in this case I did not change it, because {parentControl} makes more sense to translators than {0}.